py-polars/tests/unit/test_datelike.py

```diff
@@ -551,13 +550,16 @@ def test_date_range_lazy() -> None:
         ).date.tolist()
     )
 
-    # lazy date range with expressions
+
+@pytest.mark.parametrize("low", ["start", pl.col("start")])
+@pytest.mark.parametrize("high", ["stop", pl.col("stop")])
+def test_date_range_lazy_with_expressions(low: str | pl.Expr, high: pl.Expr) -> None:
     ldf = (
         pl.DataFrame({"start": [date(2015, 6, 30)], "stop": [date(2022, 12, 31)]})
         .with_columns(
             pl.date_range(
-                pl.col("start"),
-                pl.col("stop"),
+                low,
+                high,
                 interval="678d",
                 lazy=True,
```

Comment:
Could you remove the trailing comma in these statements? Saves a bunch of lines.